src/app/series/directives/series-podcast.component.html

```diff
@@ -49,7 +49,8 @@
       </div>
     </publish-fancy-field>
 
-    <publish-fancy-field label="Podcast Audio" [select]="audioVersionOptions" [model]="distribution" name="versionTemplateUrl" required>
+    <publish-fancy-field required label="Podcast Audio" name="versionTemplateUrl"
```

Comment:
getting `Error in ./SeriesPodcastComponent class SeriesPodcastComponent - inline template:52:60 caused by: Cannot read property '0' of undefined` when I try to create a podcast in a new series. Looks like it's coming from this line in `get versionTemplateConfirm()` : `this.audioVersionOptions.find(opt => opt[1] === url)[0];` 
When I poke around in the debugger,`url` is an empty string at that point.